Models/Interior/Panel/Instruments/kma20/kma20.xml

```diff
@@ -602,7 +622,7 @@
                 <label>Microphone: %s</label>
                 <property alias="../../../../params/knob"/>
                 <mapping>nasal</mapping>
-                <script>(arg[0] == -1) ? "COM1" : (arg[0]==1) ? "External" : "COM2"</script>
+                <script>return (arg[0] == -1) ? "COM1" : (arg[0]==1) ? "External" : "COM2";</script>
```

Comment:
Could you change this nested ternary operators to the way you do in `dme.xml`? A single ternary operator is fine, but I find nested ones not very readable. Also make sure you put a space around operators (`==`)